python/paddle/fluid/tests/unittests/test_conv2d_op.py

```diff
@@ -117,9 +119,9 @@ def test_check_grad(self):
                 set(['Input', 'Filter']),
                 'Output',
                 max_relative_error=0.02)
-        else:
-            self.check_grad(
-                set(['Input', 'Filter']), 'Output', max_relative_error=0.02)
+        place = core.CPUPlace()
```

Comment:
Syntactic sugar ?
1. `place` variable can be declared as a global variable
2. Or you should unpack this one inside a function.